app/src/main/java/com/myperishableplanner/v21001/dto/Item.kt

```diff
@@ -1,3 +1,3 @@
 package com.myperishableplanner.v21001.dto
 
-data class Item()
+data class Item(var ItemId : Int)
```

Comment:
Added parameter so that the project would build.

Reply:
Good.